app/components/footer.js

```diff
@@ -1,19 +1,97 @@
 import React from 'react'
-import { createUseStyles } from 'react-jss'
+import { createUseStyles, ThemeProvider } from 'react-jss'
+import theme from '../theme'
+import ExternalLink from './external-link.svg'
 
 const useStyles = createUseStyles({
     footer: {
+        ...theme.footer,
+        fontFamily: theme.defaultFontFamily,
         position: 'absolute',
```

Comment:
position: absolute, left: 0 and bottom: 0 should be removed.  Position absolute forces the component to be at the bottom of the viewport, when it needs to be at the bottom of the window (which could be much further down).

I noticed that the App story is using Footer so look at it there.